packages/react-native-win32/src/typings-index.ts

```diff
@@ -26,3 +26,4 @@ export * from './Libraries/Components/Touchable/TouchableWin32.Types';
 export * from './Libraries/Components/Touchable/TouchableWin32';
 export * from './Libraries/PersonaCoin/PersonaCoin';
 export * from './Libraries/PersonaCoin/PersonaCoinTypes';
+export { PlatformColor } from './Libraries/StyleSheet/NativeOrDynamicColorType.win32';
```

Comment:
If this file is flow, "PlatformColor" will just be resolved to "any" which we probably don't want. Need to add a parallel TS declaration to get the correct result (Basically what Sammy's current PR is doing for FocusManager).